CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+This repo contains an in-progress prototype of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+
+### Sign the CLA
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more information
+
+### Learn a bit about the Kubernetes Federation v2
```

Comment:
I would retitle this as "Learn About Kubernetes Federation v2" and I would also add a link to the main README where it talks about the project and concepts https://github.com/kubernetes-sigs/federation-v2#kubernetes-federation-v2.